test/integration/push_int_test.go

```diff
@@ -96,6 +96,7 @@ func (suite *PushIntegrationTestSuite) TestInitAndPush() {
 	ts.LoginAsPersistentUser()
 
 	cp = ts.SpawnWithOpts(e2e.OptArgs("push", namespace))
+	cp.SendLine("y")
```

Comment:
Do we have a record of this failing previously? What exactly was it that caused you decide to add this?

Reply:
The test would fail because it was expecting input here. I will try to remove it and see what happens.